.github/actions/install-tools/action.yaml

```diff
@@ -0,0 +1,32 @@
+name: install-tools
+description: "Install pipeline tools"
+
+runs:
+  using: composite
+  steps:
+    - uses: anchore/sbom-action/download-syft@b6a39da80722a2cb0ef5d197531764a89b5d48c3 # v0.15.8
+
+      # renovate: datasource=github-tags depName=anchore/grype versioning=semver
+    - run: "TAG= v0.105.0 curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b /usr/local/bin --tag $TAG"
+      shell: bash
+
+    - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+      with:
+        go-version: 1.22.x
+        cache-dependency-path: |
+          **/go.sum
+          **/go.mod
+
+    - name: make local bin
+      shell: bash
+      run: |
+        mkdir -p ~/.local/bin
+
+      # Once there are releases on https://github.com/defenseunicorns/maru-runner we should switch to that
+    - name: Install UDS CLI
+      shell: bash
+      env:
+        VERSION: v0.9.3
+      run: |
+        curl --location --output ~/.local/bin/uds "https://github.com/defenseunicorns/uds-cli/releases/download/${VERSION}/uds-cli_${VERSION}_Linux_amd64"
+        chmod +x ~/.local/bin/uds
```

Comment:
changing my tune, if we already bring `grype` in to `/usr/local/bin`, lets just bring `uds` there as well

Reply:
or change `grype` to install to `~/.local/bin`